packages/Tooltip/src/Tooltip.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useRef, useState } from 'react';
+import classNames from 'classnames';
+import { Position, GetVisiblePosition } from './position';
+
+import './tooltip.scss';
+
+export type Appearance = 'dark' | 'light';
+
+export interface TooltipProps extends React.ComponentProps<'div'> {
+  children: string | React.ReactNode;
+  tooltipClassName?: string;
+  content?: React.ReactNode;
+  position?: Position;
+  appearance?: Appearance;
+  maxWidth?: number;
+  arrowVisible?: boolean;
+  active?: boolean;
+}
+
+const Tooltip: React.FunctionComponent<TooltipProps> = (
+  props: TooltipProps
+) => {
+  const {
+    children,
+    content,
+    tooltipClassName,
+    position = 'top',
+    appearance = 'dark',
+    maxWidth = 200,
+    arrowVisible = true,
+    className,
+    ...rest
+  } = props;
+
+  const classes = classNames('ids-tooltip__container', className);
+
+  const defaultTooltipClasses = classNames('ids-tooltip', tooltipClassName, {
+    [`ids-tooltip--${position}`]: true,
+    'has-arrow': arrowVisible,
+    'ids-tooltip-light': appearance === 'light',
```

Comment:
On se base sur la méthodologie BEM pour la nomenclature des className, tu peux le remplacer par:
```suggestion
    'ids-tooltip--light': appearance === 'light',
```